ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ValidateSchema.java

```diff
@@ -34,7 +66,6 @@ private static void loadSchema(String name) throws SchemaNotFoundException {
       JSONObject rawSchema = Json.readJSONObject(schema);
       schemas.put(name, SchemaLoader.load(rawSchema));
     } catch (IOException e) {
-      // TODO load all existing schemas on startup
```

Comment:
excellent! we should remove the `if (!schemas.containsKey(name))` section from `getSchema` that calls `loadSchema`